<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCourseDialog</class>
 <widget class="QDialog" name="addCourseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Course</string>
  </property>
  <widget class="QLineEdit" name="courseCodeField">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelCourse">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 80, 64);
color: white</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCourseConfirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>192</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 207, 8);
color: white;</string>
   </property>
   <property name="text">
    <string>Add Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="courseNameField">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
